docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3.7'
+
+services:
+  main:
+    container_name: main
+    build:
+      context: .
+      target: development
+    volumes:
+      - .:/usr/src/app
+      - /usr/src/app/node_modules
+    ports:
+      - ${SERVER_PORT}:${SERVER_PORT}
+      - 3000:3000
+    command: npm run start:dev
+    env_file:
+      - .env
+    networks:
+      - webnet
+    depends_on:
+      - redis
```

Comment:
Esse "depends_on" de redis vai causar erro, porque não estamos criando um banco de dados de redis aqui, enão retornará um erro. Deixa somente o postgres como depends_on

Reply:
Sure, I'll fix it.